README.md

```diff
@@ -15,10 +15,12 @@ optimized for use via a Command Line Interface (CLI), with all the benefits of a
   * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
   * It comes with a **reasonable level of user and developer documentation**.
 * It is named `MyInsuRec` because it helps financial advisors to keep track of their clients and meetings.
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
```

Comment:
Perhaps it is more appropriate to redirect to <https://ay2223s1-cs2103t-w16-4.github.io/tp/>